handlers/SuiChainHandler.ts

```
@@ -272,33 +272,103 @@
+          const parsedLog = this.l1BitcoinDepositorProvider.interface.parseLog(log);
+          
+          // Check if this is the TokensTransferredWithPayload event
+          if (parsedLog.name === 'TokensTransferredWithPayload' && parsedLog.args.transferSequence) {
+            transferSequence = parsedLog.args.transferSequence.toString();
+            eventTxHash = l1Receipt.transactionHash;
+            logger.info(`Found transfer sequence ${transferSequence} in parsed log for deposit ${deposit.id}`);
+            break;
+          }
+        } catch (parseError) {
+          // This log doesn't match our interface, continue to next
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)